src/renderer/screens/nft/Gallery/TokensList/Item.js

```diff
@@ -51,7 +59,7 @@ type Props = {
   withContextMenu?: boolean,
 };
 
-const Row = ({ contract, tokenId, id, mode, account, withContextMenu = false }: Props) => {
+const NftCard = ({ contract, tokenId, id, mode, account, withContextMenu = false }: Props) => {
   const { status, metadata } = useNftMetadata(contract, tokenId);
   const { nftName } = metadata || {};
   const show = useMemo(() => status === "loading", [status]);
```

Comment:
Does this useMemo here really makes sense? I doesn't seem to me that this comparison would make much of a difference in permormance / re-renders.

Reply:
The issue this useMemo is trying to solve is that right now, each time we have a new synchronisation, we create a new array of NFTs in the `Account`, which is supposed to preserve the original ref of each of its element if they're identical to the previous synchronisation. 

It's happening here: https://github.com/LedgerHQ/ledger-live-common/blob/348b43649cdad670daafb2abe9d7cdade3251d34/src/families/ethereum/synchronisation.ts#L169
Using this method:
https://github.com/LedgerHQ/ledger-live-common/blob/master/src/bridge/jsHelpers.ts#L121

But it turns out it doesn't work as expected for now (you can test it by clicking the `✅  Synchronized` while being in a gallery, it will probably make you images blink). 
That's why we're trying to switching a PureComponent for the image, and memoizing the NftCard is also here to try to mitigate the issue as, on paper, it should almost never re-render since it's a pretty performance consuming component.